### PR TITLE
feat(client): change the layout of lowerjaw hint and test to flex

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Ref #48101 

Because the `h2` and the `p` don't swap because they aren't flex in RTL layout, I had to change the layout to flex.

image

### CODE REVIEW

File: client/src/templates/Challenges/classic/lower-jaw.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ const LowerJaw = ({
               <p>{t(sentencePicker())}</p>
             </div>
           </div>
-          <div className='hint-status fade-in' aria-hidden={isFeedbackHidden}>
+          <div className='hint-status' aria-hidden={isFeedbackHidden}>
```
Comment (Sboonny) [inline client/src/templates/Challenges/classic/lower-jaw.tsx:152]:
```suggestion
          <div className='hint-status fade-in' aria-hidden={isFeedbackHidden}>

Comment (Sboonny) [reply client/src/templates/Challenges/classic/lower-jaw.tsx]: I realized while messing with that class, I had reduced motion on the whole time

**Comment by ahmadabdolsaheb** [comment]
Thanks for looking into this.

You could include the following in this pr to cover the rest of the changes needed in the lower jaw.

- margin for the hint and status icon needs to be mirrored.
- paddings for h2 elements needs to be mirrored.
- margin for the first element of the button icons needs to be mirrored.

The spacing (margin, padding, etc) could be completely replaced with logical properties and rewritten for the RTL layout.

**Comment by Sboonny** [comment]
I think you, when you mentioned logical properties, you want me to add `margin: 5 0 0` instead of `margin-top: 5`, and `margin-right: 5`

because logical properties are confusing to beginners, I decided to add what I was going to do with the gap anyway, that will make this easier for RTL. So it becomes easier to read for beginner and I don't have to undo my work later

image

With RTL direction added to lower jar

image

**Comment by ahmadabdolsaheb** [comment]
Thanks for the hard work, 

The variables could be removed, and the spacing of the icon buttons should be adjusted.

That should conclude the RTL work on the lower jaw.

**Comment by ahmadabdolsaheb** [comment]
Solid work. Thanks @Sboonny

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**